config/jobs/kubernetes/sig-release/release-branch-jobs/1.30.yaml

```diff
@@ -0,0 +1,2473 @@
+        env:
+        - name: EXCLUDE_TYPECHECK
+          value: "y"
+        - name: EXCLUDE_GODEP
+          value: "y"
+        - name: EXCLUDE_GOLANGCI_LINT
+          value: "y"
+        - name: KUBE_VERIFY_GIT_BRANCH
+          value: release-1.30
+        - name: REPO_DIR
+          value: /workspace/k8s.io/kubernetes
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240311-b09cdeb92c-1.30
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          limits:
+            cpu: "7"
+            memory: 12Gi
+          requests:
+            cpu: "7"
+            memory: 12Gi
+        securityContext:
+          privileged: true
+  - always_run: true
+    branches:
+    - release-1.30
+    cluster: eks-prow-build-cluster
+    context: pull-kubernetes-verify-lint
+    decorate: true
+    name: pull-kubernetes-verify-lint
+    path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+      - args:
+        - verify
+        - WHAT=golangci-lint golangci-lint-pr
+        command:
+        - make
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240311-b09cdeb92c-1.30
+        name: ""
+        resources:
+          limits:
+            cpu: "5"
+            memory: 12Gi
+          requests:
+            cpu: "5"
+            memory: 12Gi
+  - always_run: true
+    branches:
+    - release-1.30
+    cluster: eks-prow-build-cluster
+    context: pull-kubernetes-linter-hints
+    decorate: true
+    name: pull-kubernetes-linter-hints
+    optional: true
+    path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+      - args:
+        - verify
+        - WHAT=golangci-lint-pr-hints
+        command:
+        - make
+        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240311-b09cdeb92c-1.30
+        name: ""
+        resources:
+          limits:
+            cpu: "5"
+            memory: 12Gi
+          requests:
+            cpu: "5"
+            memory: 12Gi
+  - always_run: false
+    branches:
+    - release-1.30
+    context: pull-kubernetes-e2e-capz-windows-1.30
+    decorate: true
+    extra_refs:
+    - base_ref: release-1.13
+      org: kubernetes-sigs
+      path_alias: sigs.k8s.io/cluster-api-provider-azure
+      repo: cluster-api-provider-azure
+    - base_ref: master
+      org: kubernetes-sigs
+      path_alias: sigs.k8s.io/cloud-provider-azure
+      repo: cloud-provider-azure
+    - base_ref: master
+      org: kubernetes-sigs
+      path_alias: k8s.io/windows-testing
+      repo: windows-testing
+      workdir: true
+    labels:
+      preset-azure-anonymous-pull: "true"
+      preset-azure-capz-sa-cred: "true"
+      preset-azure-cred-only: "true"
+      preset-capz-containerd-1-7-latest: "true"
+      preset-capz-windows-2022: "true"
+      preset-capz-windows-common-pull: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-windows-private-registry-cred: "true"
+    name: pull-kubernetes-e2e-capz-windows-master
```

Comment:
one more .. `name: pull-kubernetes-e2e-capz-windows-master` should be name: `pull-kubernetes-e2e-capz-windows-1.30`